File: Algoritmos Python/Ex13.py
nome = input("Nome do funcionário que receberá aumento:")
salario_old = float(input("Salario do funcionário:"))
salario_novo = salario_old + salario_old * 0.15
print(f'O funcinario {nome} receberá daqui em diante R${salario_novo:.2F}')

# Entrada
# A = lista1, lista2, lista3
# n = 3
# x = nome, salario_old, salario_novo
# Saida
# 1 Ajustar a reposta para NOT FOUND
# 2 Para cada indice i em A:
#   Se A[i] = x de cada lista e se lista2[i]*0,15 + lista2[i] = lista3[i], ajustar i como resposta
# 3 Retornar a resposta como saida.

File: Basic/src/Exercicios/Ex13.java
package Exercicios;

import java.text.NumberFormat;
import java.util.Scanner;

public class Ex13 {
    public static void main(String []args){
        System.out.println("Nome do funcionário que receberá aumento:");
        Scanner nome = new Scanner(System.in);
        String nomeFuncionario = nome.nextLine();

        System.out.println("Salario do funcionário:");
        Scanner sal = new Scanner(System.in);
        double salFuncionario = sal.nextDouble();
        sal.close();//o close() serve para liberar qualquer coisa armazenada para não ocupar memoria e causaar esgotamento na execução

        double salario_novo = salFuncionario + salFuncionario * 0.15;
        System.out.println("O funcinario "+nomeFuncionario+" receberá daqui em diante "+ NumberFormat.getCurrencyInstance().format(salario_novo)); //pega a configuração da maquina e devolve a moeda da lingua que a maquina esta configurada.
    }
}